src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,30 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
-public class HelloWorld {
+import java.util.Arrays;
+
+class HelloWorld {
+
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier(1,1);
+        Figure[] equal = getFig();
+        for (int i = 0; i < equal.length; i++) {
+            if (i < equal.length / 2) {
+                equal[i].draw();
+            } else {
+                System.out.println(Arrays.toString(figureSupplier.getDefaultFigure()));
+                break;
+            }
+        }
+
```

Comment:
```suggestion
        Figure [] figure = new Figure[6];
        FigureSupplier figureSupplier = new FigureSupplier();
        for (int i = 0; i < figure.length / 2; i++) {
            figure[i] = figureSupplier.getRandomFigure();
        }
        for (int i = figure.length / 2; i < figure.length; i++) {
            figure[i] = figureSupplier.getDefaultFigure();
        }
        for (Figure randomFigure: figure) {
            randomFigure.draw();
        }
    }

```

мне кажется метод getFig() тут лишний